web/src/app/documentation/components/IntegrationKeyAPI.js

```diff
@@ -1,4 +1,3 @@
-import React, { Component } from 'react'
```

Comment:
Only needed to remove `{ Component }`, still need `import React from 'react'`

Reply:
addressed, changes made.